installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -5,6 +5,17 @@ subnetIds=$2
 iamRoleARN=$3
 region=$4
 stackprefix=$5
+role=__lambda2_basic_execution_1
+iam=$stackprefix$role
+jazz_admin=$6
+
+JENKINS_JSON_PROPFILE=/home/$SSH_USER/cookbooks/jenkins/files/node/jazz-installer-vars.json
```

Comment:
$SSH_USER is not defined in the file. I doubt whether this file was the right place to make this change since this file changes the jazz-core repo in the local installer box.